packages/FlexGrid/Col/__tests__/Col.spec.jsx

```diff
@@ -46,16 +46,6 @@ describe('Col', () => {
     expect(col).toHaveProp('xsOffset', 2)
   })
 
```

Comment:
I think that unless you do mocking for the `deprecate` module we will see `console.log` output in our tests, which is not ideal.